javascript-methods/main.js

```diff
@@ -0,0 +1,59 @@
+const a = 7;
+const b = 8;
+const c = 15;
+
+const maximumValue = Math.max(a,b,c);
+console.log('value of maximumValue:', maximumValue);
+
+const heroes = ['Spider-Man', 'Iron Man', 'Wolverine', 'All-Might'];
+
+const randomNumber = Math.random() * heroes.length;
+const randomIndex = Math.floor(randomNumber);
+
+console.log('value of randomIndex:', randomIndex);
+
+const randomHero = heroes[randomIndex];
+console.log('value of randomHero:', randomHero);
+
+const library = [
+    {
+    title: 'Gone With the Wind',
+    author: 'Margaret Mitchell'
+},
+{
+    title: 'Jane Eyre',
+    author: 'Charlotte Bronte'
+},
+{
+    title: 'Pride and Prejudice',
+    author: 'Jane Austen'
+}
+];
+
+const lastBook = library.pop();
+console.log('value of lastBook:', lastBook);
+
+const firstBook = library.shift();
+console.log('value of firstBook', firstBook);
+
+var js = {
+    title: 'JavaScript for Impatient Programmers',
+    author: 'Dr. Axel Rauschmayer'
+  };
+var css = {
+    title: 'CSS Secrets',
+    author: 'Lea Verou'
+  };
```

Comment:
No `var`! This should be using `const`

Reply:
![Screenshot 2023-07-12 103009](https://github.com/dennis-pnguyen/c0723-code-solutions/assets/122412863/3d18ad37-5f52-40b3-a9d8-5bde60e1e80b)
Updated!